zubhub_frontend/zubhub/src/components/inputText/inputText.jsx

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from 'react';
+import clsx from 'clsx';
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+import 'react-toastify/dist/ReactToastify.css';
+import { handleInputTextFieldChange, handleInputTextFieldBlur } from '../../views/create_activity/createActivityScripts'
+import {
+  Box,
+  Typography,
+  FormControl,
+  FormHelperText,
+  ClickAwayListener
+} from '@material-ui/core';
+
+function InputText(props) {
+    const { classes, 
+            common_classes,
+            activity_classes, 
+            label, 
+            helperText, 
+            placeholder, 
+            vars
+            } = props 
+    const [inputTextFieldFocused, setInputTextFieldFocused] = useState(false)
+    props = {...props, inputTextFieldFocused, setInputTextFieldFocused}         
```

Comment:
Please never reassign values to the function parameters, instead create a new object